exotic/exotic.py

```diff
@@ -391,8 +393,8 @@ def getJulianTime(hdul):
 
 # Method that gets and returns the current phase of the target
 def getPhase(curTime, pPeriod, tMid):
-    phase = (curTime - tMid) / pPeriod
-    return phase - int(np.nanmin(phase))
+    phase = (curTime - tMid -0.5*pPeriod) / pPeriod % 1
+    return phase - 0.5
```

Comment:
why are you doing it like this?

Reply:
Cus it works

Reply:
Ok...what is the benefit of changing the function from what it was to what you want to make it? Speed?

Reply:
Kyle sez: "faster"